### ISSUE TITLE
No handler for insertion of document in roles API

### ISSUE THREAD

**Issue by GavinMendelGleason** [terminusdb/terminusdb#1261]
```json
{
  "api:message":"Error: can_not_insert_existing_object_with_id('terminusdb://system/data/Role/testing')\n  [42] throw(error(can_not_insert_existing_object_with_id('terminusdb://system/data/Role/testing'),_6348))\n  [40] when:trigger_first(true,'document/json':(...,...)) at /home/gavin/.swivm/versions/v8.5.8/lib/swipl/library/when.pl:100\n  [34] call(api_access_control:insert_document(...,...,'terminusdb://system/data/Role/testing')) at /home/gavin/.swivm/versions/v8.5.8/lib/swipl/boot/init.pl:499\n  [33] catch(database:call(...),fail_transaction,database:(_6500=true)) at /home/gavin/.swivm/versions/v8.5.8/lib/swipl/boot/init.pl:562\n  [32] database:with_transaction_(query_context{all_witnesses:false,authorization:'terminusdb://system/data/User/admin',bindings:[],commit_info:commit_info{author:\"admin\",message:\"API: Add Role\"},default_collection:system_descriptor{},files:[],filter:type_filter{types: ...},prefixes:_6640{'@base':\"terminusdb://system/data/\",'@schema':\"http://terminusdb.com/schema/system#\",'@type':'Context',api:'http://terminusdb.com/schema/api#',json:'http://terminusdb.com/schema/json#',owl:'http://www.w3.org/2002/07/owl#',rdf:'http://www.w3.org/1999/02/22-rdf-syntax-ns#',rdfs:'http://www.w3.org/2000/01/rdf-schema#',sys:'http://terminusdb.com/schema/sys#',vio:'http://terminusdb.com/schema/vio#',woql:'http://terminusdb.com/schema/woql#',xdd:'http://terminusdb.com/schema/xdd#',xsd:\"http://www.w3.org/2001/XMLSchema#\"},selected:[],system:system_descriptor{},transaction_objects:[...],update_guard:_6588,write_graph:system_graph{type:instance}},api_access_control:insert_document(...,...,'terminusdb://system/data/Role/testing'),_6542) at /home/gavin/dev/terminusdb/src/core/transaction/database.pl:233\n  [31] setup_call_catcher_cleanup(database:pre_transaction_tabling,database:with_transaction_(...,...,_6766),_6744,database:post_transaction_tabling) at /home/gavin/.swivm/versions/v8.5.8/lib/swipl/boot/init.pl:663\n  [27] '<meta-call>'(routes:(...,...)) <foreign>\n  [26] catch(routes:(...,...),error(can_not_insert_existing_object_with_id('terminusdb://system/data/Role/testing'),context(_6878,_6880)),routes:do_or_die(...,...)) at /home/gavin/.swivm/versions/v8.5.8/lib/swipl/boot/init.pl:562\n  [25] catch_with_backtrace(routes:(...,...),error(can_not_insert_existing_object_with_id('terminusdb://system/data/Role/testing'),context(_6952,_6954)),routes:do_or_die(...,...)) at /home/gavin/.swivm/versions/v8.5.8/lib/swipl/boot/init.pl:629\n\nNote: some frames are missing due to last-call optimization.\nRe-run your program in debug mode (:- debug.) to get more detail.\n\n",
  "api:status":"api:server_error"
}
```

**Comment by GavinMendelGleason** [comment]
Also need: 

```json
{
  "code":500,
  "message":"Unknown message: throw(error(document_not_found('User/admin'),_13176))"
}

````

**Comment by GavinMendelGleason** [comment]
Fixed in #1265
